scripts/check-gmp.gmp

```diff
@@ -1090,7 +1090,10 @@ def to_int(
     :param default: If the value is missing in the dict use this default
 
     """
-    value = source_dict.get(key, default)
+
+    value = source_dict.get(key)
+    if value in [None, ""]:
+        value = default
     if (value in ["", None]) and default_to_zero:
```

Comment:
This if branch could not be reached anymore with the newly added lines before. I can image two possible solutions
```javascript
if not value and default_to_zero:
    return 0
if not value:
    return default
```

Alternatively it would also be possible to  use only default
```javascript
if default_to_zero:
    default = 0

if not value:
    return default
```

Reply:
This if branch is reached if default is None too. In that case I want to return 0.